packages/material-ui/src/Breadcrumbs/BreadcrumbCollapsed.js

```diff
@@ -31,9 +31,26 @@ const styles = theme => ({
  * @ignore - internal component.
  */
 function BreadcrumbCollapsed(props) {
-  const { classes, ...other } = props;
+  const { classes, onClick, ...other } = props;
+
+  const handleKeyDown = e => {
+    e.preventDefault();
+    switch (e.key) {
+      case 'Space':
+      case 'Enter': {
+        onClick();
+      }
+    }
+  };
+
   return (
-    <li className={classes.root} {...other}>
+    <li
```

Comment:
What about using the button base for this? Don't we need a label to announce the action?

Reply:
Should I wrap `<li>` element into `ButtonBase` ?